### PR TITLE
Updated corbado_auth Readme to contain troubleshooting section and updated example…

### PR DESCRIPTION
**Pull request by lukaskratzel**
… description

### CODE REVIEW

File: packages/corbado_auth/README.md [hunk 1]
```diff
@@ -171,8 +172,14 @@ You can also close and reopen the app.
...
-    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/idtoken.jpeg" height="500" alt="id token">
+    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/signin-signup-image.jpg" height="500" alt="signup">
+    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/profile-token-details-image.jpg" height="500" alt="signup fingerpring">
 </p>
+
+## Troubleshooting
+
+| Type                            | Issue                     | Note                                                                                                                             |
+|---------------------------------|---------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| iOS Sim                         | Localizd string not found | For passkeys to work in the iOS simulator, you need to enable faceID first in the top menu bar under features > faceID > enrolled |
```
Comment (Hackfred) [inline packages/corbado_auth/README.md:183]: Sim == Simulator?

File: packages/corbado_auth/README.md [hunk 2]
```diff
@@ -171,8 +172,14 @@ You can also close and reopen the app.
...
+| iOS physical device             | Passkeys not working      | Passkeys do not work when running your flutter application on your physical iOS device. This has to do with the signing of your app. Rather test on a virtual iOS device or virtual/ physical Android device.                                                                                                                                 |
```
Comment (Hackfred) [inline packages/corbado_auth/README.md:184]: This comment is about the example right? (for general applications it must of course be possible to run on a physical device)

File: packages/corbado_auth/README.md (hunk 1 above)
Comment (lukaskratzel) [reply packages/corbado_auth/README.md:183]: yes

File: packages/corbado_auth/README.md (hunk 2 above)
Comment (lukaskratzel) [reply packages/corbado_auth/README.md:184]: yes. as far as I remember you told me about this because there is some issue regarding associated domains

**Pull request merged**